### ISSUE TITLE
Need documentation on how to use this 

### ISSUE THREAD

**Issue by sandip-wane** [timonwong/smart-table-improved#5]

**Comment by timonwong** [comment]
Yes, you are right... honestly, I'm not good at writing documentation in English, I'll try though.
The [wiki pages](https://github.com/timonwong/smart-table-improved/wiki) now contains a very poor documentation now, please have a look
